oshi-core/src/main/java/oshi/software/os/unix/freebsd/FreeBsdOSProcess.java

```diff
@@ -223,6 +226,30 @@ private int queryBitness() {
         return 0;
     }
 
+    @Override
+    public List<OSThread> getThreadDetails() {
+        List<OSThread> threads = new ArrayList<>();
+        String psCommand = "ps -awwxo tdname,lwp,state,etimes,systime,time,tdaddr,nivcsw,nvcsw,majflt -H";
+        if (getProcessID() >= 0) {
```

Comment:
I don't think this is nesseary. We should never have a process 0 on Linux and if we did, I'd want to get its threads!  Just add the `-p pid` to the above line.

Reply:
Ok.